### DISCUSSION TITLE
Is there any plugin or util function to display pyarrow table with rich?

### DISCUSSION THREAD

**Discussion by Roy-Kid** [Textualize/rich#3498] [Ideas]
Can we support this feature, or how to make a plugin for this general feature?

**Comment by willmcgugan** [comment]
Nothing builtin, but Rich is very extensible, so you could implement that without much effort. The following may help...

https://rich.readthedocs.io/en/latest/protocol.html
https://rich.readthedocs.io/en/latest/pretty.html#rich-repr-protocol
